### PR TITLE
Fix/1499 fix copy and move of comments

### PR DESCRIPTION
**Pull request by inkhey**
related to #1499
will be continued by #1652 fixing copy revision order between subcontent and content revision.

### CODE REVIEW

File: backend/tracim_backend/lib/core/content.py [hunk 1]
```diff
@@ -2011,3 +2098,11 @@ def generate_folder_label(self, workspace: Workspace, parent: Content = None) ->
             query = query.filter(Content.parent == parent)
 
         return _("New folder {0}").format(query.count() + 1)
+
+    def _allow_empty_label(self, content_type_slug):
```
Comment (lebouquetin) [inline backend/tracim_backend/lib/core/content.py]: typing is missing

File: backend/tracim_backend/lib/core/content.py [hunk 2]
```diff
@@ -1450,6 +1462,36 @@ def copy(
         new_file_extension: str = None,
         do_save: bool = True,
         do_notify: bool = True,
+    ) -> Content:
+        """
+        Copy nearly all content, revision included. Children not included, see
```
Comment (lebouquetin) [inline backend/tracim_backend/lib/core/content.py]: The docstring is wrong. You copy all contents, children included now.

File: backend/tracim_backend/lib/webdav/resources.py [hunk 3]
```diff
@@ -892,14 +892,13 @@ def copyMoveSingle(self, destpath, isMove):
         except ContentNotFound:
             destination_parent = None
         try:
-            new_content = self.content_api.copy(
+            self.content_api.copy(
```
Comment (lebouquetin) [inline backend/tracim_backend/lib/webdav/resources.py]: you changed the behavior without to change the name. It's dangerous. The new copy is always recursive? You need to make this information explicit somewhere (probably in the `copy()` method docstring)

File: backend/tracim_backend/lib/core/content.py (hunk 1 above)
Comment (inkhey) [reply backend/tracim_backend/lib/core/content.py]: done.

File: backend/tracim_backend/lib/core/content.py (hunk 2 above)
Comment (inkhey) [reply backend/tracim_backend/lib/core/content.py]: done.

File: backend/tracim_backend/lib/webdav/resources.py (hunk 3 above)
Comment (inkhey) [reply backend/tracim_backend/lib/webdav/resources.py]: done. It is always recursive now.

File: backend/tracim_backend/lib/core/content.py [hunk 4]
```diff
@@ -2099,7 +2099,7 @@ def generate_folder_label(self, workspace: Workspace, parent: Content = None) ->
 
         return _("New folder {0}").format(query.count() + 1)
 
-    def _allow_empty_label(self, content_type_slug):
+    def _allow_empty_label(self, content_type_slug: str):
```
Comment (lebouquetin) [inline backend/tracim_backend/lib/core/content.py]: what is the return of the function? add full typing

Comment (inkhey) [reply backend/tracim_backend/lib/core/content.py]: done.

**Pull request merged**